modules/apps/portal-search/portal-search/src/test/java/com/liferay/portal/search/internal/buffer/IndexerRequestBufferHandlerTest.java

```diff
@@ -88,7 +89,7 @@ private IndexerRequest _createIndexerRequest(Indexer<?> indexer) {
 		ReflectionTestUtil.setFieldValue(
 			indexerRequestBufferOverflowHandler,
 			"_indexerRequestBufferExecutor",
-			new IndexerRequestBufferExecutor());
+			new IndexerRequestBufferExecutorUtil());
```

Comment:
This statement for sure will fail since you removed the field here https://github.com/rafaprax/liferay-portal/pull/1290/commits/54582f6174af50eee712467ed482cac2f56839c5#diff-f8b8d44af2e772f05cd7bba0d64cf69d5236191abb0f77eb38f80d4c8cf24960L95